.github/workflows/go.yml

```diff
@@ -29,14 +29,15 @@ jobs:
       run: ./test
     - name: Run linter
       uses: golangci/golangci-lint-action@v2
+      if: ${{ matrix.go-version != '1.18.x' }}
       with:
-        version: v1.42.0
+        version: latest
```

Comment:
As in the Rust workflow, I think it's better to explicitly pin a release, so that CI doesn't randomly break whenever a linter gets stricter.